api/go/go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/lf-edge/eve/api/go
 
```

Comment:
This file must have gotten changed by accident, and is not part of this PR. This should be reverted.